_targets.R

```diff
@@ -0,0 +1,40 @@
+library(targets)
+source("1_fetch/src/fetch.R")
+source("2_process/src/process.R")
+source("3_visualize/src/visualize.R")
+source("3_visualize/src/write_diagnostics.R")
+
+tar_option_set(packages = c("tidyverse", "sbtools", "whisker"))
+
+list(
+  # Get the data from ScienceBase
+  tar_target(
+    model_RMSEs_csv,
+    fetch(filepath_out = '1_fetch/out/model_RMSEs.csv'),
+    format = "file"
+  ), 
+  # Prepare the data for plotting
+  tar_target(
+    eval_data,
```

Comment:
since this function returns a file, not a data.frame, it would be best to name the target to reflect this (see how you named the fetch one for an example!). In our group, we try to name them to make it obvious when it is a file vs an object. So, I try to name similar to the filepath without the folder structure, something along the lines of `model_summary_results_csv` would work well here.